initfiles/componentfiles/configxml/roxie.xsd.in

```diff
@@ -386,6 +386,13 @@
           </xs:appinfo>
         </xs:annotation>
       </xs:attribute>
+      <xs:attribute name="multicastTTL" type="xs:nonNegativeInteger" use="optional" default="1">
+        <xs:annotation>
+          <xs:appinfo>
+            <tooltip>The multicast TTL setting for this roxie cluster.  Zero means do not explicitly set TTL.</tooltip>
```

Comment:
Perhaps "Zero means do not explicitly set the ttl, and use the default OS setting". Also should we say "TTL (time to live)" ?